scripts/get_deployed_brackets.js

```diff
@@ -0,0 +1,34 @@
+const FleetFactory = artifacts.require("FleetFactory")
+
+const argv = require("./utils/default_yargs").option("masterSafe", {
+  type: "string",
+  describe: "Address of Gnosis Safe owning every bracket",
+  demandOption: true,
+}).argv
+
+module.exports = async (callback) => {
+  try {
+    const fleetFactory = await FleetFactory.deployed()
+    const events = await fleetFactory.getPastEvents("FleetDeployed", {
+      filter: { owner: argv.masterSafe },
+      fromBlock: 0,
+      toBlock: "latest",
+    })
+    const bracketsAsObjects = events.map((object) => object.returnValues.fleet)
+    const bracketAddresses = [].concat(...bracketsAsObjects)
+
+    // writing the brackets into a csv file
+    const createCsvWriter = require("csv-writer").createObjectCsvWriter
+    const csvWriter = createCsvWriter({
+      path: "./bracket-addresses.csv",
```

Comment:
nit: Given it's simple, I would allow this to be optionally passed in via the cmd line.